### ISSUE TITLE
Scrambled logs with current continous build

### ISSUE THREAD

**Issue by maybites** [CESNET/UltraGrid#256]
Hi

I have been using ultragrid with [MaxMSP](https://cycling74.com/downloads) with the [shell abstraction](https://github.com/jeremybernstein/shell) and this works just fine, including the printout for the console log ultragrid produces.

But with the current continuous builds the console logout get scrambled (both on windows and OSX):

```
print: UltraGrid 1.7+ (tags/continuous rev a3c3e339 built Sep 26 2022 12:19:24)
print: [1mDisplay device : [0mndi
print: [1mCapture device : [0msyphon
print: [1mAudio capture : [0mnone
print: [1mAudio playback : [0mnone
print: [1mMTU : [0m9000 B
print: [1mVideo compression: [0mnone
print: [1mAudio codec : [0mPCM
print: [1mNetwork protocol : [0mUltraGrid RTP
print: [1mAudio FEC : [0mnone
print: [1mVideo FEC : [0mnone
print: [1m[34mThis application uses NDI® available from http://ndi.tv/
print: NDI® is a registered trademark of NewTek, Inc.
print: [0mDisplay initialized-ndi
print: Video capture initialized-syphon
print: Created new RTP session with SSRC 977703241.
print: [33m[Syphon capture] FPS set to 60. Use override_fps to override if you know FPS of the server.[0m
print: [32m[Syphon capture] Using server - app: Simple Server name: [0m
print: [32m[video dec.] New incoming video format detected: 420x900 @60.00p, codec UYVY[0m
print: [32m[video dec.] Successfully reconfigured display to 420x900 @60.00p, codec UYVY[0m
print: [Syphon capture] 286 frames in 5.00013 seconds = 57.1985 FPS[0m
print: [1m[35m[ndi][0m 286 frames in 5.01932 seconds = [1m56.9798 FPS
print: [0m[0mSSRC 0x3a469149: 25984/25984 packets received (100.0000%[39m), 0 lost, max loss 0[0m
```

This only happens when I run ultragrid within the shell object, but NOT inside the terminal or the cmd-tool. 

And it doesn't happen with the most [current official release](https://github.com/CESNET/UltraGrid/releases/tag/v1.7.6).

**Comment by maybites** [comment]
Obviously this problem is very specific and not of high relevance for the development of ultragrid. I could imagine that the continuous build somehow differs from the official releases. I am just hoping for a confirmation in this direction :-)

**Comment by mpiatka** [comment]
Hi,

those strange symbols are the ANSI escape codes that are used for text formatting (colored output, boldness, etc.). It looks like the shell plugin doesn't support them. We recently switched away from the library _rang_ to a simpler solution which doesn't check the terminal emulator against a whitelist, which is probably why it doesn't happen on the stable release.

As a workaround I've just added the parameter `--param log-nocolor` which will completely disable ANSI formatting.
